app/frontend/src/utils/timeAgo.ts

```diff
@@ -1,9 +1,9 @@
 import moment from "moment";
 
-export function timeAgo(milliseconds: number) {
+export function timeAgo(date: Date) {
   //the backend fuzzes times to 15 minute intervals
-  if (new Date().getTime() - milliseconds < 15 * 60 * 1000) {
+  if (new Date().getTime() - +date < 15 * 60 * 1000) {
```

Comment:
`+date` casts to integer milliseconds since epoch, presumably? Does it work without  `+`?

Reply:
Javascript works. But Typescript will complain about date not being a number.